src/tutorials.js

```diff
@@ -3,6 +3,14 @@
 const moment = require('moment');
 
 module.exports = Object.freeze([
+  {
+    title: 'Make an Instance with Axios to Make a POST Request',
```

Comment:
For SEO purposes, it's important that both "create" and "post" are in the title. So I'd say the title should be "How to use Axios' create() Method with POST Requests"